app-doc/phrack/phrack-01.ebuild

```diff
@@ -1,19 +1,18 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 MY_P=${PN}${PV:1:2}
 DESCRIPTION="A Hacker magazine by the community, for the community"
 HOMEPAGE="http://www.phrack.org/"
-SRC_URI="http://www.phrack.org/archives/tgz/${MY_P}.tar.gz"
+SRC_URI="https://phrack.org/archives/tgz/${MY_P}.tar.gz"
```

Comment:
Since the tarballs at the new location are different from the tarballs of the same name from the old location, we run into
```
!!! Fetched file: phrack1.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      10740
!!! Expected: 10771
```

when fetching the tarball via Gentoo's mirrors.